orbit/pkg/useraction/mdm_migration_darwin.go

```diff
@@ -0,0 +1,294 @@
+//go:build darwin
+
+package useraction
+
+import (
+	"bytes"
+	"errors"
+	"fmt"
+	"os"
+	"os/exec"
+	"strconv"
+	"sync"
+	"text/template"
+	"time"
+
+	"github.com/rs/zerolog/log"
+)
+
+type swiftDialogExitCode int
+
+const (
+	primaryBtnExitCode   = 0
+	errorExitCode        = 1
+	secondaryBtnExitCode = 2
+	infoBtnExitCode      = 3
+	timeoutExitCode      = 4
+	userQuitExitCode     = 10
+	unknownExitCode      = 99
+)
+
+var mdmMigrationTemplate = template.Must(template.New("mdmMigrationTemplate").Parse(`
+## Migrate to Fleet
+
+To begin, click "Start." Your default browser will open your My Device page.
+
+{{ if .Aggresive }}You {{ else }} Once you start, you {{ end -}} will see this dialog every 15 minutes until you click "Turn on MDM" and complete the instructions.
+
+\\![Image showing the Fleet UI](http://localhost:64793/mdm-migration-screenshot-768x180@2x.jpg)
+
+Unsure? Contact {{ .OrgInfo.OrgName }} IT [here]({{ .OrgInfo.ContactURL }}).
+`))
+
+var errorTemplate = template.Must(template.New("").Parse(`
+### Something's gone wrong.
+
+Please contact your IT admin [here]({{ .ContactURL }}).
+`))
+
+// baseDialog implements the basic building blocks to render dialogs using
+// swiftDialog.
+type baseDialog struct {
+	path        string
+	interruptCh chan struct{}
+}
+
+func newBaseDialog(path string) *baseDialog {
+	return &baseDialog{path: path, interruptCh: make(chan struct{})}
+}
+
+func (b *baseDialog) CanRun() bool {
+	if _, err := os.Stat(b.path); err != nil {
+		if errors.Is(err, os.ErrNotExist) {
+			return false
+		}
+		return false
```

Comment:
Since it returns `false` regardless of the error type, you can remove the `ErrNotExist` check.